doc/sphinx/reactors.rst

```diff
@@ -195,3 +181,228 @@ temperature:
 
     m c_p \frac{dT}{dt} = - \dot{Q} - \sum_k h_k \dot{m}_{k,gen}
         + \sum_{in} \dot{m}_{in} \left(h_{in} - \sum_k h_k Y_{k,in} \right)
+
+
+Wall Interactions
+-----------------
+
+The total rate of heat transfer through all walls is:
+
+.. math::
+
+    \dot{Q} = \sum_w f_w \dot{Q}_w
+
+where `f_w = \pm 1` indicates the facing of the wall.
+
+In case of surface reactions, there is a net generation (or
+destruction) of homogeneous phase species at the wall. The molar rate of production for each species `k` on wall `w` is `\dot{s}_{k,w}` (in kmol/s/m\ :sup:`2`). The total (mass) production rate for species `k` on all walls is:
+
+.. math::
+
+    \dot{m}_{k,wall} = W_k \sum_w A_w \dot{s}_{k,w}
+
+where `W_k` is the molecular weight of species `k` and `A_w` is the area of
+each wall. The net mass flux from all walls is then:
+
+.. math::
+
+    \dot{m}_{wall} = \sum_k \dot{m}_{k,wall}
+
+
+Reactor Networks and Devices
+============================
+
+While reactors by themselves just define the above governing equations of the reactor, the time integration is performed in reactor networks. A reactor network is therefore necessary even if only a single reactor is considered.
+
+The advantage of reactor networks obviously is that multiple reactors can be interconnected. Not only mass flow from one reactor into another can be realized, but also heat can be transferred, or the wall between reactors can move. To set up a network, the following components can be defined in addition to the reactors previously mentioned:
+
+- **Reservoir**: A reservoir can be thought of as an infinitely large volume, in which all states are predefined and never change from their initial values. Typically, it represents a vessel to define temperature and composition of a stream of mass flowing into a reactor, or the ambient fluid surrounding the reactor network. Besides, the fluid flow finally finally exiting a reactor network has to flow into a reservoir. In the latter case, the state of the reservoir (except pressure) is irrelevant.
```

Comment:
Should these (and perhaps the Reactors above) link to the source code somewhere? I'm not sure if it should be the Python interface documentation or the C++, but it might be good to link the code in...